pkg/controller/direct/cloudbuild/workerpool_controller.go

```diff
@@ -106,7 +106,7 @@ func (m *model) AdapterForObject(ctx context.Context, reader client.Reader, u *u
 	if externalRef == "" {
 		id = BuildID(projectID, location, resourceID)
 	} else {
-		id, err = asID(externalRef)
```

Comment:
So the reason we call it `asID` and `asExternalRef` at the beginning is because the structured ID and `status.externalRef` are equivalent. They are two different forms of representing a ConfigConnector real ID. I feel like `asID` better reveals this relationship. 

I'm good with the other renaming, it's nit. But I think we want to align here. 

@justinsb What do you feel about this?

Reply:
I reverted the naming changes back. I'm fine with either ways, as long as we have good docs to make the functions clear.